src/taoensso/timbre/appenders/3rd_party/newrelic.clj

```diff
@@ -0,0 +1,21 @@
+(ns taoensso.timbre.appenders.3rd-party.appenders.newrelic
+  "A New Relic appender"
+  {:author "Camilo Polymeris"}
+  (:import (com.newrelic.api.agent NewRelic)
+           (java.util HashMap)))
+
+(defn newrelic-appender
+  "New Relic appender. The Java agent must be installed and configured via newrelic.yml, see
+  https://docs.newrelic.com/docs/agents/java-agent/installation/java-agent-manual-installation#h2-install-agent"
+  [& _]
+  {:enabled?   true
+   :async?     true
+   :min-level  :warn
+   :rate-limit [[100 60000]]                                ;matches New Relic's own cap
+   :output-fn  :inherit
+   :fn         (fn [data]
+                 (let [{:keys [output-fn ?err level]} data
+                       params (HashMap. {"Log level" (name level)})]
+                   (if ?err
+                     (NewRelic/noticeError ?err params)
+                     (NewRelic/noticeError (output-fn data) params))))})
```

Comment:
> the cases where I log an error (i.e. when the transaction or job failed) are exactly the same I would report to NR... why not use that?

Because you'd be complecting things that are semantically very different:

1. Reporting an API-specific error to a particular API.
2. General-purpose, multi-level logging to an arbitrary set of appenders.

So there's two questions here. First: is this the best approach to take in _your specific_ case? I'd argue not, when you can just do this with a function that sends the notification and triggers a related Timbre logging call if you like. That way you're not tying certain logging levels into a specific semantic meaning. What if you call Timbre's `error` function somewhere that has nothing to do with this NewRelic library?

Second question: is this going to be widely useful to enough other developers to warrant inclusion in Timbre itself, ongoing maintenance, etc.? Do you see many other users wanting this particular behaviour?

Does that make sense / seem reasonable?